agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/sampling/Samplers.java

```diff
@@ -28,11 +28,11 @@ public static Sampler getSampler(
     }
```

Comment:
can we remove the `Configuration.SamplingPreview`?

can you check for any other references to `SamplingPreview`?